ddsp/core.py

```diff
@@ -208,10 +208,14 @@ def gradient_reversal(x):
 
 
 # Unit Conversions -------------------------------------------------------------
-def midi_to_hz(notes: Number) -> Number:
+def midi_to_hz(notes: Number, zero_silence: bool = False) -> Number:
   """TF-compatible midi_to_hz function."""
```

Comment:
Can you add a fuller doc string to this? (following the style of other doc strings)

Reply:
Added a fuller doc string for midi_to_hz.